src/views/AuthoritiesSearch/AuthoritiesSearch.js

```diff
@@ -192,9 +223,55 @@ const AuthoritiesSearch = ({
     ...options,
   ];
 
+  const getActionItem = ({ id, icon, messageId, onClickHandler, isDisabled = false }) => {
+    return (
+      <Button
+        buttonStyle="dropdownItem"
+        id={id}
+        disabled={isDisabled}
+        onClick={onClickHandler}
+      >
+        <Icon
+          icon={icon}
+          size="medium"
+          iconClassName={css.actionIcon}
+        />
+        <FormattedMessage id={messageId} />
+      </Button>
+    );
+  };
+
+  const triggerQuickExport = async (sendCallout) => {
+    if (instancesQuickExportInProgress) return;
+
+    setInstancesQuickExportInProgress(true);
+
+    try {
+      const instanceIds = Object.keys(selectedRows).slice(0, 100);
```

Comment:
Same here - authorities instead of instanced.
Also, let's ask Khalilah about export limit amount